app/src/androidTest/java/com/duckduckgo/app/browser/BrowserTabViewModelTest.kt

```diff
@@ -2442,6 +2442,21 @@ class BrowserTabViewModelTest {
         assertCommandIssued<Command.CheckSystemLocationPermission>()
     }
 
+    @Test
+    fun whenDomainRequestsSitePermissionAndUserAllowedSessionPermissionAndThenPermissionIsAllowed() = coroutineRule.runBlocking {
```

Comment:
Nit: I think the last `And` in the test name is unnecessary and it should be `whenDomainRequestsSitePermissionAndUserAllowedSessionPermissionThenPermissionIsAllowed`

Reply:
AND you are right about that